scripts/build.sh

```diff
@@ -975,6 +975,12 @@ custom_rootfs_setup() {
 		echo '[Install]' >> $file
 		echo 'WantedBy=multi-user.target' >> $file
 	done
+	for file in ${CHROOT_BASEDIR}/tmp/systemd/multi-user.target.wants/*.service; do
```

Comment:
So you want to exclude every file but rrdcached.service? Curious why?

@kmoore134 you being the shell expert can think of a smarter way to do this? Maybe find with `-name \! "rrdcached.service -delete`?

Reply:
If a unit is in `multi-user.target.wants`, there is no way to disable it (you run `service disable nslcd`, but it is still started after a reboot). With `rrdcached` it is not the case, we don't want to disable it.